tests/in/image-1d.wgsl

```diff
@@ -0,0 +1,13 @@
+// Check that we request the `Sample1D` SPIR-V capability if we use `texture_1d`.
```

Comment:
Do we need to create a separate test file for each small feature being tested? It would be easier to navigate the tests if you can at least scroll through them.
Perhaps, we could just add an image to `image.wgsl` or `globals.wgsl` instead?

Reply:
In general, yes, I'd love to gather tests into coherent groups. But the comment just below this explains why this specific test can't be combined with others.

Reply:
For example, `image.wgsl` does include a `texture_1d`. But it also includes a `texture_storage_1d`, which is why it didn't catch the problem.

Reply:
how about globals.wgsl then?